### PR TITLE
add Intel's new media-driver

### PR DESCRIPTION
**Pull request by lrusak**
This adds a package for intels new media-driver which will eventually replace intel-vaapi-driver for newer intel platforms

>The Intel(R) Media Driver for VAAPI is a new VA-API (Video Acceleration API) user mode driver supporting hardware accelerated decoding, encoding, and video post processing for GEN based graphics hardware.

I did not include this by default as it is quite large and requires manual user interaction to use it anyways.
For example:
```
LIBVA_DRIVER_NAME="iHD" /usr/lib/kodi/kodi.bin
```

### CODE REVIEW

File: packages/multimedia/gmmlib/package.mk [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PKG_NAME="gmmlib"
+PKG_VERSION="50707fb182bc8fc8f9711d2c0da96f6ded1b8ef4"
+PKG_SHA256="73c8895e3d5c54c8a44682c69db5c1a40da5500dacc07c89cb520d2b21626b05"
+PKG_ARCH="x86_64"
+PKG_LICENSE="MIT"
+PKG_SITE="https://01.org/linuxmedia"
+PKG_URL="https://github.com/intel/gmmlib/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_LONGDESC="$PKG_NAME: The Intel(R) Graphics Memory Management Library provides device specific and buffer management for the Intel(R) Graphics Compute Runtime for OpenCL(TM) and the Intel(R) Media Driver for VAAPI."
```
Comment (MilhouseVH) [inline packages/multimedia/gmmlib/package.mk]: Nit: replace `$PKG_NAME` with `gmmib`, as you have it in the package below?

Comment (lrusak) [reply packages/multimedia/gmmlib/package.mk]: whoops, thanks

**Comment by lrusak** [comment]
merge?

File: packages/multimedia/gmmlib/package.mk [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+PKG_NAME="gmmlib"
+PKG_VERSION="50707fb182bc8fc8f9711d2c0da96f6ded1b8ef4"
+PKG_SHA256="73c8895e3d5c54c8a44682c69db5c1a40da5500dacc07c89cb520d2b21626b05"
+PKG_ARCH="x86_64"
+PKG_LICENSE="MIT"
+PKG_SITE="https://01.org/linuxmedia"
+PKG_URL="https://github.com/intel/gmmlib/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_LONGDESC="gmmlib: The Intel(R) Graphics Memory Management Library provides device specific and buffer management for the Intel(R) Graphics Compute Runtime for OpenCL(TM) and the Intel(R) Media Driver for VAAPI."
+PKG_TOOLCHAIN="cmake"
```
Comment (CvH) [inline packages/multimedia/gmmlib/package.mk]: needed if only cmake is at the package ?

File: packages/multimedia/media-driver/package.mk [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+PKG_NAME="media-driver"
+PKG_VERSION="82c0e6f7f6fd920218031a1d202b4ae06e3cf148"
+PKG_SHA256="6f689d174a7a9597bc1d4b368530f822f29c2a26200c44bead6d885754b0e4b8"
+PKG_ARCH="x86_64"
+PKG_LICENSE="MIT"
+PKG_SITE="https://01.org/linuxmedia"
+PKG_URL="https://github.com/intel/media-driver/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain libva libdrm gmmlib"
+PKG_LONGDESC="media-driver: The Intel(R) Media Driver for VAAPI is a new VA-API (Video Acceleration API) user mode driver supporting hardware accelerated decoding, encoding, and video post processing for GEN based graphics hardware."
+PKG_TOOLCHAIN="cmake"
```
Comment (CvH) [inline packages/multimedia/media-driver/package.mk]: needed if only cmake is at the package ?

File: packages/multimedia/gmmlib/package.mk (hunk 2 above)
Comment (lrusak) [reply packages/multimedia/gmmlib/package.mk]: removed

File: packages/multimedia/media-driver/package.mk (hunk 3 above)
Comment (lrusak) [reply packages/multimedia/media-driver/package.mk]: removed

**Pull request merged**